File: JAVA/Find maximum volume of a cuboid.java
class Solution {
    double maxVolume(double perimeter, double area) {
        double length = (perimeter - Math.sqrt(perimeter * perimeter - 24 * area)) / 12;
        double height = (perimeter / 4) - (2 * length);
        double volume = length * length * height;
        return volume;
    }
}

File: C++/Find maximum volume of a cuboid.cpp
class Solution {
  public:
    double maxVolume(double perimeter, double area) {
        double p=perimeter,a=area;
        double l=(p/2-sqrt(p*p/4-6*a))/6;
        return l*l*(p/4-2*l);
    }
};

